04_numeronym/alextmz/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"unicode/utf8"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	r := make([]string, len(vals))
+	for k, v := range vals {
+		r[k] = numeronym(v)
+	}
+	return r
+}
+
+func numeronym(s string) string {
+	if len(s) < 4 {
```

Comment:
This exhibits a similar problem mentioned in my earlier review, and is not doing what you think it's doing. See my comment on the test cases for some examples of where this will fail.

As described in https://blog.golang.org/strings, a string is effectively a read-only slice of bytes, not runes, so the length of a string is not the number of runes, but the number of bytes.

Reply:
Yep, I corrected this - the `len` usage was quite naive and the source of the issue.